TLM/TLM/UI/SubTools/TimedTrafficLightsTool.cs

```diff
@@ -99,9 +100,37 @@ public TimedTrafficLightsTool(TrafficManagerTool mainTool)
         }
 
         public override void OnPrimaryClickOverlay() {
-            if (HoveredNodeId <= 0 || nodeSelectionLocked) {
+            if (HoveredNodeId <= 0 || nodeSelectionLocked || !Flags.MayHaveTrafficLight(HoveredNodeId)) {
                 return;
             }
+            bool ctrlDown = Input.GetKey(KeyCode.LeftControl) || Input.GetKey(KeyCode.RightControl);
+            if(ctrlDown) {
+                AutoTimedTrafficLights.ErrorResult res = AutoTimedTrafficLights.Setup(HoveredNodeId);
+                if (res != AutoTimedTrafficLights.ErrorResult.Success) {
+                    string message;
+                    switch (res) {
+                        case AutoTimedTrafficLights.ErrorResult.NoJunction:
+                            message = "Dialog.Text:This is not a junction";
+                            break;
+                        case AutoTimedTrafficLights.ErrorResult.NoNeed:
+                            message = "Dialog.Text:There is no need";
+                            break;
+                        case AutoTimedTrafficLights.ErrorResult.TTLExists:
+                            message = "Dialog.Text:Timed traffic lights already exists";
+                            break;
+                        default: //Unreachable code
+                            message = $"error = {res}";
+                            break;
+                    }
+                    message = Translation.TrafficLights.Get("Dialog.Text:Default timed traffic lights not createed because : ") +
```

Comment:
Translation key is too long and detailed. Change to shorter and more specific, something like:
"Dialog.Text:Auto TL create failed because" and move your text into the English and English UK. There should not be space before and after colon. Typo in "Createed".